packages/react-charting/src/components/GaugeChart/GaugeChart.base.tsx

```diff
@@ -393,6 +422,8 @@ export class GaugeChartBase extends React.Component<IGaugeChartProps, IGaugeChar
       return {
         d,
         segmentIndex: this._isRTL ? segments.length - 1 - index : index,
+        startAngle: prevAngle - (segment.size / (total - minValue)) * Math.PI,
+        endAngle,
```

Comment:
why is this code change needed

Reply:
Added `startAngle` and `endAngle` properties here as a part of the arc object returned by `_processProps()`. 
They are needed in the render function to specify the start and stop angles of the gradient

Reply:
does this work in RTL mode properly. RTL mode can be toggled from the top right corner on the storybook.